src/main/java/org/generationcp/middleware/service/api/phenotype/PhenotypeSearchObservationDTO.java

```diff
@@ -1,78 +1,218 @@
 package org.generationcp.middleware.service.api.phenotype;
 
 import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonView;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import liquibase.util.StringUtils;
+import org.generationcp.middleware.api.brapi.v2.germplasm.ExternalReferenceDTO;
+import org.generationcp.middleware.api.brapi.v2.observation.ObservationDto;
+import org.generationcp.middleware.service.api.BrapiView;
 import org.generationcp.middleware.service.api.study.SeasonDto;
 import org.pojomatic.Pojomatic;
 
 import java.util.Date;
+import java.util.List;
+import java.util.Map;
 
 public class PhenotypeSearchObservationDTO {
+
```

Comment:
The "searchDTO" in class name is inappropriate as to how we use it (not as DTO used in search function). 
Further, check if we can delete this class altogether and replace its usage with `org.generationcp.middleware.api.brapi.v2.observation.ObservationDto`

Reply:
As discussed, this might be risky so it's better to address this during the tech period. Logged as IBP-5303.